aclk/agent_cloud_link.c

```diff
@@ -955,16 +955,18 @@ static void aclk_main_cleanup(void *ptr)
         QUERY_THREAD_WAKEUP;
         // Send a graceful disconnect message
         time_t time_created = now_realtime_sec();
+        usec_t time_created_offset_usec = (now_realtime_usec() % USEC_PER_SEC);
```

Comment:
This will be non-monotonic if the second rolls over between the two calls.

Reply:
Will fetch usecs only and do the math